securedrop/translations/ar/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,1350 @@
+"<a href=\"tor2web.org\">Tor2Web</a> وسيط يعين على النفاذ إلى الخدمات "
+"والمواقع المنشورة من شبكة تور (ذات اسم النطاق onion.) بدون استخدام تور. "
+"و&nbsp;هو في الأساس مصمم لتيسير الوصول إلى م النشر المُجهَّل."
+
+#: source_templates/tor2web-warning.html:5
+msgid ""
+"Tor2Web only protects publishers, not readers. If you upload documents to us "
+"using Tor2Web, you are <strong>not anonymous</strong> and could be "
+"identified by your ISP or the Tor2Web proxy operators. Additionally, since "
+"Tor2Web sites typically do not use HTTPS, it is possible that your "
+"connection could be MITM'ed by a capable adversary."
+msgstr ""
+"و في حين أن مجهولية الناشر حفوظة بسبب استخدامهم تور، فإنّ Tor2Web لا تحمي "
+"مجهوليّة القراء. فإذا أودعت لدينا مستندات بوساطة Tor2Web فإن هويتك <strong>لا "
+"تكون مُجهّلة</strong> و&nbsp;يمكن لمقدِّم خدمة الإنترنت لك أو لمُشغل Tor2Web "
+"التعرّف على هويتك و&nbsp;ربطها بنشاطك. \" \"بالإضافة إلى ما سبق فبسبب أن "
+"خدمات Tor2Web عادة لا تطبِّق بروتوكول HTTPS مبدئيًا فإنّ اتصالك قد يمكن اعتراضه "
+"من قِبَل خصم متمكّن."
+
+#: source_templates/tor2web-warning.html:6
+msgid ""
+"If you want to submit information, you are <strong>strongly advised</strong> "
+"to install <a href=\"torproject.org/download.html.en\">Tor</a> and use it to "
+"access our site safely and anonymously."
+msgstr ""
+"لغرض إيداع المعلومات فإننا <strong>ننصح بشدّة</strong> باستخدام <a href="
+"\"torproject.org/download.html.en\">متصفح تور</a> من أجل النفاذ إلى موقعنا "
+"على بأمان و&nbsp;مجهولية."
+
+#: source_templates/use-tor-browser.html:3
+msgid "You Should Use Tor Browser"
+msgstr "الأحرى بك استخدام متصفح تور"
+
+#: source_templates/use-tor-browser.html:4
+msgid ""
+"If you are not using Tor Browser, you <strong>may not be anonymous</strong>."
+msgstr "في حال عدم استخدام متصفح تور فإنّ <strong>هويتك لن تكون مُجهّلة</strong>."
+
+#: source_templates/use-tor-browser.html:5
+msgid ""
+"If you want to submit information to SecureDrop, we <strong>strongly advise "
+"you</strong> to install Tor Browser and use it to access our site safely and "
+"anonymously."
+msgstr ""
+"لغرض إيداع المعلومات فإنّنا <strong>ننصح بشدّة</strong> باستخدام متصفح تور من "
+"أجل النفاذ إلى هذا الموقع بأمان و&nbsp;مجهوليّة."
+
+#: source_templates/use-tor-browser.html:6
+msgid ""
+"Copy and paste the following address into your browser and follow the "
+"instructions to download and install Tor Browser:"
+msgstr ""
+"انسخ العنوان التالي و&nbsp;ألصقه في متصفحك و&nbsp;اتّبع الإرشادات لتنزيل "
+"و&nbsp;تنصيب متصفّح تور:"
+
+#: source_templates/use-tor-browser.html:9
+msgid ""
+"If there is a chance that downloading the Tor Browser raises suspicion and "
+"your mail provider is less likely to be monitored, you can send a mail to "
+"<pre>gettor@torproject.org</pre> and a bot will answer with instructions."
+msgstr ""
+"إذا كان زيارة موقع تور لتنزيل المتصفح مثير للريبة، وحساب بريدك الإلكتروني "
+"غير مراقب يمكنك مراسلة <pre> 1gettor@torproject.org</pre> 2 وسوف تتلقى "
```

Comment:
Ah, this looks like a mistake: `<pre> 1gettor@torproject.org</pre> 2` -> `<pre> gettor@torproject.org</pre>`. Should we fix this in Weblate and then sync again to add that change to this PR @dachary?